### PR TITLE
reduce error messages like: 'No handlers could be found for logger'

### LINKED ISSUE #362: neurolucida: No handlers could be found for logger
`neurolucida` tries to write to a logger, and when there isn't one, this message appears:

> No handlers could be found for logger "neurom.io.neurolucida"

To reproduce:

``` python
from neurom import fst
nrn = fst.load_neuron('test_data/neurolucida/bio_neuron-001.asc')
```

### PR DESCRIPTION
**Pull request by mgeplf**
* add NullLogger as per:
    https://docs.python-guide.org/writing/logging/#logging-in-a-library
* checked that if logging is setup, the logs are still propagated
* hat time @genric
* fixes #362

### CODE REVIEW

**Comment by mgeplf** [comment]
ping

**Comment by wizmer** [comment]
From: https://pythonhosted.org/logutils/libraries.html
> to avoid this, the library developer simply needs to instantiate a NullHandler and add it to the top-level logger of the library module or package.
I haven't tried but this suggests that we should be able to put the NullHandler in the top level __init__.py and be fine with it. Have you tried it ?

**Comment by mgeplf** [comment]
Didn't try that, does seem like a more elegant solution

File: tox.ini [hunk 1]
```diff
@@ -19,10 +19,13 @@ commands = nosetests
 [testenv:lint]
 basepython = python3
 deps =
-    pycodestyle==2.3.1
+    pycodestyle
+    # need to pin b/c https://github.com/PyCQA/astroid/issues/650
```
Comment (wizmer) [inline tox.ini]: Ah!

Comment (mgeplf) [reply tox.ini]: yeah...we'll have to delete this once it's fixed upstream :(

**Pull request merged**